src/lib/components/events/event-item.svelte

```diff
@@ -47,9 +47,7 @@
         </h2>
 
         <p class="event-location">
-          {info.location === 'Discord' || info.location === 'Zoom'
-            ? `Hosted on ${info.location}`
-            : info.location}
+          {info.location?.trim() || 'TBD'}
```

Comment:
In which case wouldnt this be redundant?

Reply:
Updated per conversation with Ethan, UI will do formatting and parse will handle null checks

Reply:
i see